### PR TITLE
test: add tests to WipeTokenAccountPrecompileSuite

### LINKED ISSUE #12458: Implement negative test cases for wipe

### PR DESCRIPTION
**Pull request by mustafauzunn**
**Description**:

add tests to WipeTokenAccountPrecompileSuite

**Related issue(s)**:

Fixes #12458 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/WipeTokenAccountPrecompileSuite.java [hunk 1]
```diff
@@ -16,21 +16,78 @@
...
+    private static final String SECOND_ACCOUNT = "anybodySecond";
+    public static final String WIPE_KEY = "wipeKey";
+    private static final String MULTI_KEY = "purpose";
+    public static final int GAS_TO_OFFER = 1_000_000;
+    public static final String WIPE_FUNGIBLE_TOKEN = "wipeFungibleToken";
+    public static final String WIPE_NON_FUNGIBLE_TOKEN = "wipeNonFungibleToken";
+
+    public static final String THRESHOLD_KEY = "TreshKey";
+
+    private static final KeyShape TRESHOLD_KEY_SHAPE = KeyShape.threshOf(1, ED25519, CONTRACT);
```
Comment (MrValioBg) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/WipeTokenAccountPrecompileSuite.java:90]:
```suggestion
    private static final KeyShape THRESHOLD_KEY_SHAPE = KeyShape.threshOf(1, ED25519, CONTRACT);
```

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/WipeTokenAccountPrecompileSuite.java [hunk 2]
```diff
@@ -50,4 +107,259 @@ public boolean canRunConcurrent() {
     public List<HapiSpec> getSpecsInSuite() {
         return List.of();
```
Comment (MrValioBg) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/WipeTokenAccountPrecompileSuite.java:108]: Shouldn't we add the new HAPI in the getSpecsInSuite method?

**Review by stoyanov-st — APPROVED** [review]
LGTM!

**Review by tinker-michaelj — APPROVED** [review]
LGTM, ty @mustafauzunn !

**Pull request merged**